R/makephyloseqFromTreeSummarizedExperiment.R

```diff
@@ -229,15 +229,15 @@ setMethod("makePhyloseqFromTreeSummarizedExperiment", signature = c(x = "ANY"),
 }
 
 # In phyloseq, tips and rownames must match
-.get_rowTree_for_phyloseq <- function(x, tree_name){
+.get_rowTree_for_phyloseq <- function(x, tree.name){
     # Check if the rowTree's tips match with rownames:
     # tips labels are found from rownames
-    if( any(!( rowTree(x, tree_name)$tip) %in% rownames(x)) ){
+    if( any(!( rowTree(x, tree.name)$tip) %in% rownames(x)) ){
         # If rowtree do not match, tree is pruned
-        x <- .get_x_with_pruned_tree(x, tree_name)
+        x <- .get_x_with_pruned_tree(x, tree.name)
     }
     # Get rowTree
-    phy_tree <- rowTree(x, tree_name)
+    phy_tree <- rowTree(x, tree.name)
     # Convert rowTree to phyloseq object
     phy_tree <- phyloseq::phy_tree(phy_tree)
```

Comment:
In this kind of internal functions, it is not necessary to change parameter names. We should keep the code as stable as possible and modify it only when necessary.

This makes maintaining harder since there are large changes that do not really affect the functionality. Also it is harder to keep track the functional changes

--> so my suggestion is to keep internal functions and variables as they are, and only  modify those parameters that are visible for user.